### ISSUE TITLE
Feature request: Skip babel transpilation

### ISSUE THREAD

**Issue by S0ulDrag0n** [codesandbox/sandpack#137]
# Tell us how you think we can improve Sandpack

To give some context, I have additional library files in a `public` folder that is meant to be statically served. Currently, I am passing it to the `Sandpack` component via the `files` prop. However, these files have already been transpiled and passing them through the `files` prop seems to make the babel transpiler do it again. You would think that running it through would not have much impact but it turns out it just generates errors.

## Packages affected

- [ ] sandpack-client
- [x] sandpack-react

## What is this feature?

I am proposing that a flag be added to the `SandpackFile` type to skip babel transpilation.

## How would your idea work?

```
<Sandpack
  files={{
    '/public/jquery.min.js': {
      code: '/public/jquery.min.js',
      ignore: true,
    }
  }}
/>
```

**Comment by DeMoorJasper** [comment]
@danilowoz this is definitely doable. However this code would still be passed through our custom transformer that collects dependencies and rewrites import statements  which uses acorn and it super-fast.

Or maybe it should be more of a isolated module that can never have any dependencies?

Sounds very similar to `https://github.com/codesandbox/sandpack/issues/119` which we already support in the bundler

**Comment by S0ulDrag0n** [comment]
> Or should it be more of an isolated module that can never have any dependencies?

I think there's a chance that may be the case. Perhaps there should be another flag for skipping that as well?

**Comment by DeMoorJasper** [comment]
Yeah probably, for your example you can probably use `externalResources` for now, but we've added this feature request to the backlog and will get to it once we've reworked the bundler to support this.

**Comment by amareshsm** [comment]
@danilowoz  Is this issue still open?

**Comment by danilowoz** [comment]
Yeah, this's a valid issue. However, it's not a priority now. 
Please thumbs up the main comment for those who are interested in this feature.

**Comment by iampava** [comment]
+1 for skipping babel transpilation.
